Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MKMapViewTests.swift
+//  SwifterSwift
+//
+//  Created by Hannes Staffler on 24.01.19.
+//  Copyright © 2019 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(MapKit)
+import MapKit
+
+import struct CoreLocation.CLLocationCoordinate2D
+
+final class MKMapViewTests: XCTestCase {
+
+    func testRegisterAndDequeue() {
+        guard #available(iOS 11.0, *, tvOS 11.0, *, macOS 10.13) else {
+            return
+        }
+
+        let mapView = MKMapView()
+        let annotation = MKPlacemark(coordinate: CLLocationCoordinate2DMake(0, 0))
```

Comment:
`CLLocationCoordinate2DMake` is soooo Objective-C 😜
```suggestion
        let annotation = MKPlacemark()
```
This change will also mean that you can remove `import struct CoreLocation.CLLocationCoordinate2D`